sim/priest/shadow/shadow_priest_test.go

```diff
@@ -128,6 +128,8 @@ func TestShadow(t *testing.T) {
 			EPReferenceStat: proto.Stat_StatSpellPower,
 			StatsToWeigh:    Stats,
 		},
+		core.GetTestBuildFromJSON(proto.Class_ClassPriest, 7, 60, "../../../ui/shadow_priest/builds", "phase_7", ItemFilters, proto.Stat_StatSpellPower, Stats),
+		core.GetTestBuildFromJSON(proto.Class_ClassPriest, 8, 60, "../../../ui/shadow_priest/builds", "phase_8", ItemFilters, proto.Stat_StatSpellPower, Stats),
```

Comment:
I presume this is all I need in the way of tests?

Reply:
Yep that should work! It pulls all of the settings from the JSON